app/uk/gov/hmrc/agentclientrelationships/services/EmailService.scala

```diff
@@ -16,68 +16,107 @@
+            )
+          )
+        )
+    }
+  }
+
+  def sendExpiredEmail(invitation: Invitation)(implicit ec: ExecutionContext): Future[Boolean] = {
     implicit val hc: HeaderCarrier =
       HeaderCarrier(extraHeaders = Seq("Expired-Invitation" -> s"${invitation.invitationId}"))
-    sendEmail(invitation, "client_expired_authorisation_request")
+    logger.info(s"[EmailService] sending email for invitation ${invitation.invitationId} that has expired")
+    emailConnector.sendEmail(
+      EmailInformation(
+        to = Seq(invitation.agencyEmail),
+        templateId = "client_expired_authorisation_request",
+        parameters = Map(
+          "agencyName" -> invitation.agencyName,
+          "clientName" -> invitation.clientName,
+          "expiryDate" -> DateTimeHelper.displayDate(invitation.expiryDate),
+          "service"    -> messagesApi(s"service.${invitation.service}")
+        )
+      )
+    )
   }
 
-  def sendAcceptedEmail(invitation: Invitation)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] =
+  def sendAcceptedEmail(invitation: Invitation)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] =
     sendEmail(invitation, "client_accepted_authorisation_request")
 
-  def sendRejectedEmail(invitation: Invitation)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] =
+  def sendRejectedEmail(invitation: Invitation)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] =
     sendEmail(invitation, "client_rejected_authorisation_request")
 
   def sendEmail(invitation: Invitation, templateId: String)(implicit
     hc: HeaderCarrier,
     ec: ExecutionContext
-  ): Future[Unit] =
+  ): Future[Boolean] =
     for {
       agentDetailsDesResponse <- agentAssuranceConnector.getAgentRecordWithChecks(Arn(invitation.arn))
       emailInfo = emailInformation(templateId, invitation, agentDetailsDesResponse)
-      _ <- emailConnector.sendEmail(emailInfo)
-    } yield ()
+      emailResult <- emailConnector.sendEmail(emailInfo)
+    } yield emailResult
 
   private def emailInformation(
     templateId: String,
     invitation: Invitation,
     agentDetailsDesResponse: AgentDetailsDesResponse
-  ) = {
-    val dateFormatter: DateTimeFormatter =
-      DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK)
-
-    EmailInformation(
-      Seq(agentDetailsDesResponse.agencyDetails.agencyEmail),
-      templateId,
-      Map(
-        "agencyName" -> agentDetailsDesResponse.agencyDetails.agencyName,
-        "clientName" -> invitation.clientName,
-        "expiryDate" -> invitation.expiryDate.format(dateFormatter),
-        "service"    -> invitation.service,
-        "additionalInfo" -> {
-          if (invitation.isAltItsa)
-            s"You must now sign ${invitation.clientName} up to Making Tax Digital for Income Tax."
-          else ""
-        }
-      )
+  ) = EmailInformation(
+    Seq(agentDetailsDesResponse.agencyDetails.agencyEmail),
```

Comment:
we have these details in the invitation, so why call DES?

Reply:
This was already in place for the accept/reject, I wasn't using it for my warning/expired but yeah it's a fair point. Val mentioned it too so I'll remove it 👍